streams/us_tubi.m3u

```diff
@@ -1,6 +1,6 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="EstrellaNews.us",Estrella News (720p)
-https://csm-e-eb.csm.tubi.video/csm/live/247083838.m3u8
+https://csm-e-eb.csm.tubi.video/csm/extlive/tubiprd01,Cloudfront-Estrella.m3u8
```

Comment:
link redirects to TS playlist files, however master https://lnc-estrella.tubi.video/playlist.m3u8 works.